templates/notifications/email_mention.html.j2

```diff
@@ -0,0 +1,2 @@
+<div>You were mentioned by <span class='fw-bold'>@{{ username }}</span> in a discussion of <a href='{{url_for("main.runs", **{ "test_id": test_id, "additionalRuns[]": run_id })}}'>{{ build_id }}#{{ build_number }}</a>. </div>
```

Comment:
keep in mind that bootstrap css classes like `fw-bold` and such will not be rendered in the sent email, so this would need `style` attribute to embolden the text instead or full html template document